book/src/chap01.asciidoc

```diff
@@ -40,7 +40,7 @@ Para evitar ese problema, se recomienda que comience ejecutando Julia en un nave
 
 En ((("JuliaBox")))((("recursos en línea", "JuliaBox")))el navegador, puede ejecutar Julia en https://www.juliabox.com[JuliaBox]. No se requiere instalación: simplemente abra su navegador y entre a la página de JuliaBox, inicie sesión y comience a programar (consulte <<juliabox>>).
 
-Julia _REPL_ (Read–Eval–Print Loop) ((("REPL (Read-Eval-Print Loop)")))es un programa que lee y ejecuta código de Julia. Puede iniciar REPL abriendo un terminal en JuliaBox y escribiendo *+julia+* en la línea de comando. Cuando comienza, debería ver una salida como esta:
+Julia _REPL_ (de "Read–Eval–Print Loop" que se traduce en "Leer-Evaluar-Imprimir Bucle") ((("REPL (Read-Eval-Print Loop)")))es un programa que lee y ejecuta código de Julia. Puede iniciar REPL abriendo un terminal en JuliaBox y escribiendo *+julia+* en la línea de comando. Cuando comienza, debería ver una salida como esta:
```

Comment:
Bucle de "Leer-Evaluar-Imprimir" is the correct conjugation

Reply:
I guess that your'e suggesting that the content in parentheses be changed from "de 'Read–Eval–Print Loop' que se traduce en 'Leer-Evaluar-Imprimir Bucle'" that I use something like "de 'Read–Eval–Print Loop' que se traduce en 'Bucle de Leer-Evaluar-Imprimir'".  Is that right?

Reply:
yeah